### ISSUE TITLE
Very ugly lines around the Old Camp and rice fields

### ISSUE THREAD

**Issue by Spatzendame** [SaiyansKing/GD3D11#114]
These ugly li´nes are already visible in the original game, but when using the dx11 patch they are even more visible. Can you completely remove them?

https://imgur.com/rXs3xw0

https://imgur.com/tEO5uqC

https://imgur.com/5mYd7Qp

https://imgur.com/U0wzR57

**Comment by postm1** [comment]
It is mesh wholes, on dx11 u can see such places more easier than on dx7, it is not render's issue.

**Comment by Spatzendame** [comment]
Hi proxin, yes it is mesh holes, but when it is easier to see on dx11 than
on dx7, then it is a renderer issue. I do not mean to fix this mesh
completely but just to avoid seeing it more easier than on dx7.
Greetings

Am Mo., 24. Apr. 2023 um 12:27 Uhr schrieb proxin ***@***.***

**Comment by postm1** [comment]
Yeas, I know, but no fix fo this I guess ;)

**Comment by Spatzendame** [comment]
Are you a developer of the dx11 renderer?
I really hope this will be fixed because it is very annoying issue.
Or can someone make a world mesh fix?

Am Di., 25. Apr. 2023 um 08:22 Uhr schrieb proxin ***@***.***

**Comment by postm1** [comment]
Nope, I help sometimes a little ;) The lines on the screens are very huge, never seen them like that btw. Better to fix mesh, but it can do only the mod's maker.
